tc-tests-utils.sh

```diff
@@ -492,6 +492,47 @@ do_deepspeech_binary_build()
     deepspeech
 }
 
+# Hack to extract Ubuntu's 16.04 libssl 1.0.2 packages and use them during the
+# local build of Python.
+#
+# Avoid (risky) upgrade of base system, allowing to keep one task build that
+# builds all the python packages
+maybe_ssl102_py37()
```

Comment:
That's a bit of a hack, but it allows us to stay on 14.04 to produce all binaries. Seems less maintainance burden than having to make a specific, 16.04-based, build path for Python 3.7